homeassistant/components/somfy/coordinator.py

```diff
@@ -0,0 +1,71 @@
+"""Helpers to help coordinate updated."""
```

Comment:
Please exclude all new untested modules from coverage calculation in `.coveragerc`.